base/multidimensional.jl

```diff
@@ -142,11 +146,13 @@ module IteratorsMD
     # nextind and prevind with CartesianIndex
     function Base.nextind(a::AbstractArray{<:Any,N}, i::CartesianIndex{N}) where {N}
         iter = CartesianIndices(axes(a))
-        return CartesianIndex(inc(i.I, first(iter).I, last(iter).I))
+        valid, I = inc(i.I, first(iter).I, last(iter).I)
```

Comment:
Should this really be called `valid`? How about just `_`?

Reply:
```suggestion
        _, I = inc(i.I, first(iter).I, last(iter).I)
```